src/module/actor/character/document.ts

```diff
@@ -742,9 +742,10 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
             rollOptionsAll[`attribute:${key}:mod:${mod}`] = true;
         }
 
-        for (const key of SKILL_ABBREVIATIONS) {
+        for (const key of SKILL_LONG_FORMS) {
```

Comment:
Can we rename to `SKILL_SLUGS` now? :grin:

Reply:
Not yet but that would be great to do after the next step. I want to keep the diffs as small as I can for now to not overwhelm the review process.